langchain/chains/chat_vector_db/base.py

```diff
@@ -90,7 +91,7 @@ def _call(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         else:
             new_question = question
         docs = self.vectorstore.similarity_search(
-            new_question, k=self.top_k_docs_for_context, **vectordbkwargs
+            new_question, k=self.top_k_docs_for_context, search_distance=self.search_distance, **vectordbkwargs
```

Comment:
we dont need to put it in here, we can just put it in vectordbkwargs, right?

Reply:
You are right. Doing some updates to use `vectordbkwargs` instead